### PR TITLE
General updates

### PR DESCRIPTION
**Pull request by danielchatfield**

### CODE REVIEW

File: app/models/upload.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import DS from 'ember-data';
+
+export default DS.Model.extend({
+  attachment: DS.attr('file'),
+  filename: DS.attr('string'),
+  createdAt: DS.attr('date')
```
Comment (priyeshpatel) [inline app/models/upload.js]: Should we call this `dateUploaded` for consistency with `dateCreated`?

Or change the other to `createdAt` too?

**Comment by priyeshpatel** [comment]
LGTM :+1:

File: app/models/upload.js (hunk 1 above)
Comment (danielchatfield) [inline app/models/upload.js]: I was going to call it `dateCreated` but forgot to change it.

**Comment by danielchatfield** [comment]
@priyeshpatel I've changed the attribute

**Pull request merged**